src/main/java/org/opentest4j/AssertionFailedError.java

```diff
@@ -76,23 +75,12 @@ public AssertionFailedError(String message, Object expected, Object actual, Thro
 	}
 
 	private AssertionFailedError(String message, ValueWrapper expected, ValueWrapper actual, Throwable cause) {
-		super();
-		this.message = message;
+		super((message == null || message.trim().length() == 0) ? "" : message);
```

Comment:
Please update the JavaDoc for all public constructors explaining that a `null` or _empty_ message will be stored as a _blank_ message.